example/cr-detailed.yaml

```diff
@@ -143,4 +89,4 @@ spec:
 
     # Pass the JVM options to controller and segmentstore
     controllerJvmOptions: ["-XX:MaxDirectMemorySize=1g"]
-    segmentStoreJVMOptions: ["-XX:MaxDirectMemorySize=1g"]
\ No newline at end of file
+    segmentStoreJVMOptions: ["-Xmx4g", "-XX:MaxDirectMemorySize=12g"]
```

Comment:
Again these values should be what will work for a basic minimum pravega deployment and not values configured for performance.